og.module

```diff
@@ -306,3 +313,64 @@ function og_invalidate_group_content_cache_tags(EntityInterface $entity) {
     Cache::invalidateTags(Cache::buildTags('og-group-content', $tags));
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_insert() for OgRole entities.
+ */
+function og_og_role_insert(OgRoleInterface $role) {
+  // Create actions to add or remove the role, except for the required default
+  // roles 'member' and 'non-member'. These cannot be added or removed.
+  if ($role->getRoleType() === OgRoleInterface::ROLE_TYPE_REQUIRED) {
+    return;
+  }
+
+  $add_id = 'og_membership_add_single_role_action.' . $role->getName();
+  if (!Action::load($add_id)) {
+    $action = Action::create([
+      'id' => $add_id,
+      'type' => 'og_membership',
+      'label' => t('Add the @label role to the selected members', ['@label' => $role->label()]),
```

Comment:
Unfortunately, we cannot use `$role->label()` here because we might face this case: 2 roles have the same name:

- `node-software-tester`, with label "Software Tester"
- `node-hardware-tester`, with label "Hardware Tester"

The role that comes first will steal the label of the action. If  `node-software-tester` is created first, the action label will be "Add the Software Tester role to the selected members". But this action is used also for `node-hardware-tester`. In the context of the 2nd role, we'll still show the label from the 1st role. But the action will run correctly, only the label of the action will be wrong.

For this reason we should use `$role->getName()` instead. I know, it's a machine-name but, at least, is not misleading.

I've created a PR for this change https://github.com/ec-europa/og/pull/2.